### PR TITLE
Azgame cog

### PR DESCRIPTION
**Pull request by jasperdanan**
Too old. merge.

### CODE REVIEW

File: azgame/azgame.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
+
+import random
+import discord
+from discord.ext import commands
+import sqlite3
+import asyncio
+import os.path
+from cogs.utils.dataIO import dataIO
+
+db_name = "logic.db"
```
Comment (Chovin) [inline azgame/azgame.py:10]: what's this?

File: azgame/azgame.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+            await self.bot.say("Ending the game, the winning word was {}.\nBlame {}!".format(self.memory["solution"],
+              ctx.message.author.mention))
+            self.memory["playing"] = False
+            del self.memory["solution"]
+            del self.memory["wordlist"]
+
+    # @self.bot.command
+    async def begin_az(self):
+        """Loads the word database into memory."""
+        c = self.bot.db.cursor()
```
Comment (Chovin) [inline azgame/azgame.py:55]: bot doesn't have a db

File: azgame/azgame.py [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+        print("Loading words from database...")
```
Comment (Chovin) [inline azgame/azgame.py:56]: Let's keep terminal spam to a minimum

File: azgame/azgame.py [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+            self.memory["wordlist"].append(row[0])
+        print("Loaded.")
+        self.memory["solution"] = random.choice(self.memory["wordlist"])
+        print("Solution word is {}.".format(self.memory["solution"]))
+        self.memory["playing"] = True
+
+    async def play_az(self, word):
+        """Tries to redefine the current word range by seeing if a given word is in the range.
+        If the solution is given, the winner is recognized and the game ends.
+        Usage: $az word"""
```
Comment (Chovin) [inline azgame/azgame.py:70]: please use `[p]` to denote the bot's prefix. not everybody uses `$`

**Review by Chovin — CHANGES_REQUESTED** [review]

**Comment by jasperdanan** [comment]
Cog written with Discord.py directly. Was not made for redDiscord-bot

**Comment by Chovin** [comment]
This is a repo for cogs for the Red-DiscordBot. Adding this "cog" will conflict with Red's downloader. Please open a different repo if you'd like to write your own bot

**Pull request closed without merging**